### PR TITLE
[DOC] Add new property setting fast_double_writer to install documentation 

### LINKED ISSUE #4513: [DOC] New system property: opensearch.xcontent.fast_double_writer
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [x] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** 
 
Provide the documentation for recently introduced new system property `opensearch.xcontent.fast_double_writer` that allows to configure using fast double serialization features which provides better performance for serialization of floating point numbers.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

Covered in issue: https://github.com/opensearch-project/OpenSearch/issues/7822

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Adds new property setting `opensearch.xcontent.fast_double_writer=[true|false]` to installation documentation

### Issues Resolved
#4513 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@andrross @mgodwan Thank you for providing the information about the new property setting. Please review this PR and let me know if any changes are needed.

File: _install-and-configure/install-opensearch/index.md [hunk 1]
```diff
@@ -102,9 +102,9 @@ Do not declare the same JVM options in multiple locations because it can result
...
 
+Property | Description
+:---------- | :-------- 
+`opensearch.xcontent.string.length.max=<value>`  |	By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.  | 
+`opensearch.xcontent.fast_double_writer=[true|false]` | By default, OpenSearch uses the floating point parsing capability built in to the Java Runtime Environment. Set this value to true to configure OpenSearch to use the [FastDoubleParser](https://github.com/wrandelshofer/FastDoubleParser) library to parse floating point numbers, which has shown to be faster in many cases. The property can be set to determine whether to use standard Java code to write floats/doubles (default) or use the Schubfach algorithm, which is faster. The latter approach may lead to small differences in the precision of the float/double that is written to the JSON output. Default is `false`.  |
```
Comment (andrross) [inline _install-and-configure/install-opensearch/index.md:110]: I'd shorten this to:

> By default, OpenSearch serializes floating point numbers using the default implementation provided by the Java Runtime Environment. Set this value to true to use the Schubfach algorithm, which is faster but may lead to small differences in precision. Default is `false`.

Comment (vagimeli) [reply _install-and-configure/install-opensearch/index.md]: Great suggested rewrite. Accepted and updated.

File: _install-and-configure/install-opensearch/index.md [hunk 2]
```diff
@@ -102,9 +102,9 @@ Do not declare the same JVM options in multiple locations because it can result
...
+OpenSearch has a number of system properties, listed in the following table, that you can specify in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS` using `-D` command line argument notation.
 
+Property | Description
+:---------- | :-------- 
+`opensearch.xcontent.string.length.max=<value>`  |	By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.  | 
+`opensearch.xcontent.fast_double_writer=[true|false]` | By default, OpenSearch serializes floating point numbers using the default implementation provided by the Java Runtime Environment. Set this value to true to use the Schubfach algorithm, which is faster but may lead to small differences in precision. Default is `false`. |
```
Comment (cwillum) [inline _install-and-configure/install-opensearch/index.md:110]:
```suggestion
`opensearch.xcontent.fast_double_writer=[true|false]` | By default, OpenSearch serializes floating point numbers using the default implementation provided by the Java Runtime Environment. Set this value to `true` to use the Schubfach algorithm, which is faster but may lead to small differences in precision. Default is `false`. |
```

File: _install-and-configure/install-opensearch/index.md [hunk 3]
```diff
@@ -102,9 +102,9 @@ Do not declare the same JVM options in multiple locations because it can result
...
-
-- `opensearch.xcontent.string.length.max=<value>`
-
-By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.
+OpenSearch has a number of system properties, listed in the following table, that you can specify in `config/jvm.options` or `OPENSEARCH_JAVA_OPTS` using `-D` command line argument notation.
 
+Property | Description
+:---------- | :-------- 
+`opensearch.xcontent.string.length.max=<value>`  |	By default, OpenSearch does not impose any limits on the maximum length of the JSON string fields. To protect your cluster from potential distributed denial-of-service (DDoS) or memory issues, you can set the `opensearch.xcontent.string.length.max` system property to a reasonable limit (the maximum is 2,147,483,647), for example, `-Dopensearch.xcontent.string.length.max=5000000`.  |
```
Comment (cwillum) [inline _install-and-configure/install-opensearch/index.md:109]: Is this contradictory?

"OpenSearch does not impose any limits on the maximum length of the JSON string fields."

and

"(the maximum is 2,147,483,647)"

Or are these statements referring to two different maximums?

**Review by cwillum — APPROVED** [review]
LGTM

**Review by natebower — COMMENTED** [review]
@vagimeli Just a couple minor changes. Thanks!

File: _install-and-configure/install-opensearch/index.md (hunk 3 above)
Comment (vagimeli) [reply _install-and-configure/install-opensearch/index.md:109]: It's referring to the reasonable limit max
